### ISSUE TITLE
Transcriber Registration - Transcriber agreement issue

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#557]
I thought it linked back to the FR1 sign-up?

A better work-around might be to have an agreement statement that simply says 'The Transcriber Agreement is in development.  You will be consulted on this in due course.'?

**Comment by Captainkirkdawson** [comment]
EricD said

I thought Kirk had blocked anyone registering in FR2, so I don't know how she registered yesterday.  She has not registered in FR1 so will not be able to upload files.

But you have a very good point here.  Perhaps we should ask the volunteer which county that they are interested in, but then only assign them to the "Training" Syndicate.  The person running that Syndicate can then assess their experience and do the training, knowing which Syndicate to pass them to afterwards.  The Syndicate that they have asked for can be notified that someone is on the way, but not have to do the assessment.

We designed this system because we want to move to online data entry and get many more transcribers.  There will be an online training package, so the idea that I have put forward in the first paragraph will work when we get to online data entry.

KD said
There was no block to the registration software itself.

There just are no links to it on the website as far as I know. Those I removed at your and Pat's request.

The only way for someone to volunteer in FR2 is by using the contact us link and using the volunteer subject.

That just sends a contact report to the volunteer coordinator who I believe is currently EricD

Its a pretty obscure way of volunteering IMHO
